### DISCUSSION TITLE
请问下，在哪里能看到不同service的socket可以sendall什么结构的数据呢？如详情

### DISCUSSION THREAD

**Discussion by youngfreeFJS** [YueChen-C/py-ios-device#111] [Q&A]
<img width="799" alt="image" src="https://user-images.githubusercontent.com/46949537/236984955-6eb01d08-46d7-42ca-a116-fdf51a49827f.png">

如：SERVICE_NAME = 'com.apple.dt.simulatelocation'
清理时传：LOCATION_STRUCT.build({'status': 1})
设置时传：location_byte = LOCATION_STRUCT.build({
                                                  'status': 0,
                                                  'latitude': str(latitude),
                                                  'longitude': str(longitude)
                                              })
在哪里可以看到，不同服务与预期结构体的映射关系呢？

**Comment by youngfreeFJS** [comment]
@YueChen-C 感谢🙏

**Comment by youngfreeFJS** [comment]
似乎是reademe所说的： Apple instruments protocol
但没找到官网文档地址

**Reply by YueChen-C** [reply]
这还没有到 instruments 上面呢，这还仅是 Lockdown 服务相关内容，网上有很多资料

**Comment by YueChen-C** [comment]
大部分情况下，都可以通过中间人抓包的方式查看交互内容

**Comment by youngfreeFJS** [comment]
@YueChen-C 收到，感谢，找到了
